cephfs/file_test.go

```diff
@@ -512,8 +494,8 @@ func TestFallocate(t *testing.T) {
 
 	// Allocate space - default case, mode == 0.
 	t.Run("modeIsZero", func(t *testing.T) {
-		switch serverVersion {
```

Comment:
this is fine, but we could keep `serverVersion` just call the new `CurrentCephVersionString` method in init.

Reply:
Good suggestion, updated.